pages/linux/loginctl.md

```diff
@@ -0,0 +1,24 @@
+# loginctl
+
+> Manage the systemd login manager.
+> More information: <https://manned.org/loginctl>.
+
+- Print the current sessions:
+
+`loginctl list-sessions`
+
+- Print all the properties of a specific session:
+
+`loginctl show-session {{sessionId}} --all`
```

Comment:
Argument placeholders should be snake_case. Please also change this for the other occurences.
```suggestion
`loginctl show-session {{session_id}} --all`
```

Reply:
made the changes for all other occurrences